cmd/siac/rentercmd.go

```diff
@@ -540,6 +540,9 @@ func renterfileslistcmd() {
 				redundancyStr = "-"
 			}
 			uploadProgressStr := fmt.Sprintf("%.2f%%", file.UploadProgress)
+			if _, err := os.Stat(file.LocalPath); os.IsNotExist(err) && file.Redundancy < 1 {
+				uploadProgressStr = fmt.Sprintf("Local File Lost")
```

Comment:
Maybe we should print something like "File probably unrecoverable". It is very unlikely that it is recoverable but not 100% certain.

Reply:
I was trying to be concise as possible so that it doesn't look to weird in the table otherwise I would include both statements, "Local File Lost and File Probably Unrecoverable"

In this case I think it makes more sense to tell the user that the Local File is Lost since that is the failure more we are trying to catch and we would want the user to know the issue with the local file.  If the user just sees that the File is probably unrecoverable that wouldn't direct them to look at their locally file immediately.

Reply:
Maybe we should just change the header of the table to `Unrecoverable` and the entries to `yes` and `no`. That should be pretty concise.

Reply:
I can do that.  I think I'd want to add another column too then for `Local File Present` and have a `true` or `false`.  

That way the <1 redundancy would trigger the `unrecoverable` and the checking for the file on disk would trigger the `Local File Present`

Reply:
Doing it this way should also resolve #3067

Reply:
'Local File Present' is also a bit long. Maybe just 'On Disk' or something like that? We can probably play around with the name and see what looks best.

Reply:
:+1: